### PR TITLE
Feature/1 create base structure

### LINKED ISSUE #1: Create base structure
- Include np to publish npm package: https://www.freecodecamp.org/news/how-to-publish-packages-to-npm-the-way-the-industry-does-things-2077ec34d7e8/

### PR DESCRIPTION
**Pull request by Nasdan**
Close #1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  "keywords": [
+    "form",
+    "validation",
+    "validation",
+    "validate",
+    "form",
+    "async",
+    "validation",
+    "sync",
+    "validation"
```
Comment (brauliodiez) [inline package.json]: Remove repeated keywords

File: rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      format,
+      // globals: {}, // Necessary for externals libraries and umd format
+    },
+    external: Object.keys(pkg.peerDependencies || {}),
+    plugins: [
+      resolve(),
+      commonjs(),
+      typescript({
+        tsconfig: 'tsconfig.json',
+        rollupCommonJSResolveHack: true, // To be compatible with commojs plugin
```
Comment (brauliodiez) [inline rollup.config.js]: Typo in commment

File: package.json (hunk 1 above)
Comment (Nasdan) [reply package.json]: Done

File: rollup.config.js (hunk 2 above)
Comment (Nasdan) [reply rollup.config.js]: Done

**Pull request merged**